src/test/java/com/amazonaws/gurureviewercli/adapter/ArtifactAdapterTest.java

```diff
@@ -64,4 +63,52 @@ public void test_zipAndUpload_happyCaseSourceOnly() throws Exception{
         Assertions.assertNotNull(metaData.getSourceKey());
     }
 
+
+    @Test
+    public void test_zipAndUpload_happyCaseGitFilesOnly() throws Exception {
```

Comment:
Is there a test case that covers the case when the option "scan only version controlled files" is not set?

Reply:
Good point. Let me add one.